src/components/home/Event.jsx

```diff
@@ -7,7 +7,7 @@ const Event = ({ start, end, location, name, descripton, color }) => {
   return (
     <div className="bg-ewb-blue-100 flex w-full font-bold">
       <div
-        className={`text-4xl text-white text-center w-1/6 aspect-square flex flex-col items-center justify-center ${COLORS[color].bg}`}
```

Comment:
use breakpoint like "w-1/2 md: w-1/6" so the with will remain 1/6 on screen larger than md and it will be 1/2 on small screen